### PR TITLE
Adds documentation skeleton

### PR DESCRIPTION
**Pull request by RodolfoFerro**
Work done so far for #8

### CODE REVIEW

**Review by edublancas — APPROVED** [review]

**Comment by edublancas** [comment]
great, please now add the readthedocs.yml so we deploy to readthedocs.org, check this repo as reference: https://github.com/ploomber/jupysql

**Comment by edublancas** [comment]
Ok, so I added the project to readthedocs.org and added you as maintainer, I think you should be able to add the config and deploy

**Comment by RodolfoFerro** [comment]
I created a PR (#10) for this, once it is merged I can configure the deploy (I've accepted the invite for access to the rtd project).

**Pull request merged**
